core/chains/evm/assets/currencies.go

```diff
@@ -0,0 +1,117 @@
+package assets
+
+import (
```

Comment:
nit: should we rename this file to eth? It only contains eth implementation right now. Also this file is large enough to discourage others from adding more code here that is not directly related to eth